external_APIs/WeatherExternalAPI.js

```diff
@@ -0,0 +1,55 @@
+const axios = require("axios");
+const {
+  APPID,
+  FORECAST_URL,
+  weekday,
+  getHourDivisibleByThree,
+} = require("../common/index");
+const { getDailyWeatherExternalAPIResponse } = require("../DTOs/index");
+
+class WeatherExternalAPI {
+  FORECAST_URL;
+  APPID;
+  currentHour;
+  weekday;
+  constructor() {
+    this.FORECAST_URL = FORECAST_URL;
+    this.APPID = APPID;
+    this.currentHour = getHourDivisibleByThree(new Date().getHours());
```

Comment:
currentHour should not be part of this class it should be evaluated inside functions
why we are doing is because suppose we have stored object of WeatherExternalAPI  in memory and whenever we call getDailyWeather it will always give us same currenthour which is wrong